src/BPEssentials/Commands/Moderation/Warns/WarnRemove.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.ExtensionMethods.Warns;
+using BrokeProtocol.Collections;
+using BrokeProtocol.Entities;
+using System.Collections.Generic;
+using static BPEssentials.ExtensionMethods.Warns.ExtensionPlayerWarns;
```

Comment:
Static using statements are prone for future issues and code readability. Change if possible.
```suggestion

```